src/pages/api/auth/callback.ts

```diff
@@ -0,0 +1,27 @@
+import type { APIRoute } from "astro";
+import { supabase } from "../../../lib/supabase";

```

Comment:
I would suggest the following:
```suggestion
import { supabase } from "@/lib/supabase";
```

This will make it clearer and easier to understand the location of the file.